### ISSUE TITLE
Return correct exit code from runE2E function

### ISSUE THREAD

**Issue by dezsiszabi** [playwright-community/playwright-ng-schematics#6]
Hi,

Currently there is a bug in the following file: https://github.com/playwright-community/playwright-ng-schematics/blob/main/src/builders/playwright/index.ts

The `runE2E` function always returns the following object: `{ success: true }`.

It should do something like this: 

```
function runE2E(options: Options, _context: BuilderContext): BuilderOutput {
  const ret = spawnSync('npx playwright test', buildArgs(options), {
    cwd: process.cwd(),
    stdio: 'inherit',
    shell: true,
  });

  return { success: ret.status === 0 };
}
```

Otherwise the exit code of running `ng e2e` is always going to be 0, even if there are failing playwright tests. This causes tests to never fail when run during CI.

Please fix as soon as possible, if it's not too much to ask.

Please look at other schematics projects, for example:

https://github.com/webdriverio/webdriverio-schematics/blob/main/src/builders/webdriverio/index.ts#L101-L122

https://github.com/nightwatchjs/nightwatch-schematics/blob/main/src/builders/nightwatch/index.ts#L64-L69

https://github.com/puppeteer/puppeteer/blob/main/packages/ng-schematics/src/builders/puppeteer/index.ts#L214-L220

**Comment by jfgreffier** [comment]
My bad, indeed this would make CI unusable.

Fixed in 1.0.3

Thanks for the clear bug report and the suggestions !

**Comment by dezsiszabi** [comment]
Thank you @jfgreffier for the quick turnaround.
Upgraded my project and it works now. Thanks again!
